chandra_aca/transform.py

```diff
@@ -215,6 +216,18 @@ def count_rate_to_mag(count_rate):
     return mag
 
 
+def snr_mag_for_t_ccd(t_ccd, ref_mag, ref_t_ccd, scale_4c=None):
+    """
+    Given a tccd, solve for the magnitude that has the same expected signal
+    to noise as ref_mag / ref_tccd.
+
```

Comment:
For the record and future reference, can you add a derivation of the equation?

Reply:
Sure.  I was planning to put something here and also to reference your simplified form.  For the derivation, do you want all the steps or just the source/setup equation?

Reply:
Whatever you think will be enough to satisfy our future selves.